doc/ChangeLog

```diff
@@ -1,4 +1,265 @@
 ===============================================================
+Tag name: ctsm5.3.014
+Originator(s): multiple (see contributors below)
+Date: Mon 02 Dec 2024 09:50:09 AM MST
+One-line Summary: Bring in several fixes for testing in the previous cesm3_0_beta03/04 tags
+
+Purpose and description of changes
+----------------------------------
+
+Fix a list of issues mostly for testing that came in with the cesm3_0_beta03 and cesm3_0_beta04 tags
+for the science "chill" deadline bringing in the baseline science capabilities needed for the cesm3_0 release.
+
+List of things:
+
+- New polarcap grid
+- Fix use_init_interp for several resolutions that had
```

Comment:
This sentence seems incomplete.

Reply:
Fixed by adding a single word to finish it.